brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-891 Fix: Post layout: On responsive device it taking extra spacing on right.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-891

### Screenshots

- fixed Ss - https://d.pr/v/tMaoDi

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any Post layout in ss I used grid.
- Set columns to 3 on tablet and mobile.
- And check by inspecting for each article on tablet/mobile width should be 33.33%, not 31%. 
- Setting up 33.33% for tablet and mobile ( style.scss ) resolved the issue of extra space on responsive devices.

### Checklist:
